src/DynamicMvvm.Uno.WinUI/Extensions/DispatcherQueueExtensions.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/main/License.md
+// See reference: https://github.com/CommunityToolkit/WindowsCommunityToolkit/blob/main/Microsoft.Toolkit.Uwp/Extensions/DispatcherQueueExtensions.cs
+
+using System;
+using System.Threading.Tasks;
+using Windows.Foundation.Metadata;
+using Microsoft.UI.Dispatching;
+using System.Runtime.CompilerServices;
+
+namespace MessageDialogService.Extensions
+{
+	internal static class DispatcherQueueExtensions
+	{
+		/// <summary>
+		/// Indicates whether or not <see cref="DispatcherQueue.HasThreadAccess"/> is available.
+		/// </summary>
+		private static readonly bool IsHasThreadAccessPropertyAvailable = ApiInformation.IsMethodPresent("Windows.System.DispatcherQueue", "HasThreadAccess");
```

Comment:
Are there cases when this can be false on the platforms we target?

Reply:
After checking a bit more, I think this bool isn't necessary here. It was used with the `DispatcherQueue` (`Windows.System`) that isn't used in the Windows App SDK, our `DispatcherQueue` is from `Microsoft.UI.Dispatching`. I will try something with the other implementation I made of this extension ([MessageDialogService](https://github.com/nventive/MessageDialogService/blob/master/src/MessageDialog.Uno.WinUI/Extensions/DispatcherQueueExtensions.cs)) to make sure removing this bool wouldn't break the `DispatcherQueue`.